index.html

```diff
@@ -110,18 +114,6 @@
     margin-top: 0;
   }
 }
-@media (orientation: portrait) {
-  #screen canvas {
-    height: auto;
-    width: 100%;
-  }
-}
-@media (orientation: landscape) {
-  #screen canvas {
-    height: 100%;
-    width: auto;
-  }
-}
```

Comment:
Why removing these styles? They make sure that the screen aspect ratio is preserved on mobile, whatever the screen orientation is.